src/main/java/jpo/sdw/depositor/DepositorProperties.java

```diff
@@ -66,9 +66,14 @@ void initialize() {
       }
 
       if (getSubscriptionTopics() == null || getSubscriptionTopics().length == 0) {
-         String topics = String.join(",", DEFAULT_SUBSCRIPTION_TOPICS);
-         logger.info("No Kafka subscription topics specified in configuration, defaulting to {}", topics);
-         subscriptionTopics = DEFAULT_SUBSCRIPTION_TOPICS;
+         // get environment variable SDW_SUBSCRIPTION_TOPIC
+         String topics = System.getenv("SDW_SUBSCRIPTION_TOPIC");
+         subscriptionTopics = topics.split(",");
```

Comment:
If topics is null, which we check for just below, this will throw an exception with the `.split`

Reply:
Lets get this resolved and I will merge it after.

Reply:
Updated to split topics after the null check